packages/descendants/src/DescendantsReducer.ts

```diff
@@ -0,0 +1,136 @@
+import { ComponentProps, Reducer } from 'react';
+import isEqual from 'lodash/isEqual';
+import isUndefined from 'lodash/isUndefined';
+
+import { findDescendantIndexWithId } from './utils/findDescendantWithId';
```

Comment:
not blocking: Is there an advantage of importing these individually over importing everything from one index file?

Reply:
Not really, no. I just didn't make an index file 🙃  Will fix